src/platform/intel/cavs/platform.c

```diff
@@ -139,15 +133,6 @@ int platform_init(struct sof *sof)
 	/* Set CPU to max frequency for booting (single shim_write below) */
 	trace_point(TRACE_BOOT_PLATFORM_CPU_FREQ);
 
-#if CONFIG_TIGERLAKE
-	/* prevent DSP Common power gating */
-	pm_runtime_get(PM_RUNTIME_DSP, PLATFORM_PRIMARY_CORE_ID);
-
-#if CONFIG_DSP_RESIDENCY_COUNTERS
```

Comment:
I think this configuration option is now unused and can be removed?

Reply:
@lyakh  It's still implemented in XTOS pm_runtime.c and would be removing a feature from XTOS builds. So keeping in at least for now.